src/style-guide/README.md

```diff
@@ -1257,6 +1257,47 @@ While attribute values without any spaces are not required to have quotes in HTM
 ```
 </div>
 
+### Avoid toRefs with props <sup data-p="b">strongly recommended</sup>
```

Comment:
I understand the desire to single out `props` for special attention, as it does seem likely to be the most common example of this problem. But I wonder whether the more general case, of any reactive object with optional properties, would be worth acknowledging?